lisa/testsuite.py

```diff
@@ -350,6 +351,11 @@ def simple_requirement(
             disk = schema.DiskOptionSettings()
         node.disk = disk
         node.disk.data_disk_count = search_space.IntRange(min=min_data_disk_count)
+    if min_nic_count or network_interface:
+        if not network_interface:
+            network_interface = schema.NetworkInterfaceOptionSettings()
+            network_interface.nic_count = search_space.IntRange(min=min_nic_count)
```

Comment:
This line should be moved out of the `if not network_interface`.

Reply:
Wouldn't this always overwrite it to `min_nic_count`?

Reply:
If the min_nic_count is specified, it's priority is higher. If it's not necessary, it can be removed from parameters. The behavior is the same as disk.count. BTW, the variable should be named as `count`. The `data_disk_count` is data disk, so it needs to specify that.